src/Planet/planet_parameters.jl

```diff
@@ -55,12 +55,11 @@ Planet.grav(ps::AbstractEarthParameterSet) = 9.81
 Planet.year_anom(ps::AbstractEarthParameterSet) = 365.26 * Planet.day(ps)
 Planet.orbit_semimaj(ps::AbstractEarthParameterSet) = 1 * astro_unit()
 Planet.tot_solar_irrad(ps::AbstractEarthParameterSet) = 1362.0
-Planet.epoch(ps::AbstractEarthParameterSet) = 2451545.0 * Planet.day(ps)
+Planet.epoch(ps::AbstractEarthParameterSet) = DateTime(2000,01,01,11,58,56,816)
```

Comment:
This is also a breaking change..

Reply:
@tapios This is what you had in mind, right?

Reply:
Yes, something like this. I suspect you cannot have functions in the new ClimaParameters. So it may be preferable to have epoch as a string parameter, with value "2000-01-01T11:58:56", which then is parsed to Julian date inside insolation.